django/contrib/auth/management/__init__.py

```diff
@@ -66,7 +66,7 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
     except LookupError:
         return
 
-    if not router.allow_migrate(using, Permission):
+    if not router.allow_migrate_model(using, Permission):
```

Comment:
Should `allow_migrate_model()` be public API if Django makes use of it in its own signal handlers (seems third-party code may have a similar need)?